frontend/javascripts/oxalis/model/sagas/isosurface_saga.ts

```diff
@@ -617,6 +623,16 @@ function* loadPrecomputedMeshForSegmentId(
 
   const version = meshFile.formatVersion;
   try {
+    const isosurfaceExtraInfo = yield* call(getIsosurfaceExtraInfo, segmentationLayer.name, null);
+    const mappingName = meshFile.mappingName == null ? isosurfaceExtraInfo.mappingName : null;
```

Comment:
I would have expected this to always be `isosurfaceExtraInfo.mappingName`. If the active mesh file was computed for a mapping, but the mapping is not active, the segments in the viewports won't correspond to the segments in the mesh file. That's why before loading a precomputed mesh, the user is prompted to activate the corresponding mapping (if there is one). Does that make sense?

Reply:
Good point, this was a bit fuzzy here. I assumed that the mappings would match, but now I added a proper assertion to make sure that the caller didn't request something weird. I also tried to simplify the logic a bit. Please have a look at my latest commit :)